universal_notifications/management/commands/parse_pending_sms.py

```diff
@@ -13,6 +13,9 @@ class Command(BaseCommand):
     help = 'Check if message parsed'
 
     def handle(self, *args, **options):
+        # TODO: configurable? Now user needs to remember to run it every minute via cron
+        # maybe there should be some cron file created for UN by default?
+        # or maybe "ago" should be simply removed?
```

Comment:
it is "older than 1 minute", not "newer than 1 minute", so there is no problem here